server/controllers/v1/tests/comment.test.ts

```diff
@@ -83,7 +83,7 @@ describe('Test v1 - Comment Controller', () => {
       expect(result.data!.result!.body).toBe(newBody);
     });
 
-    it('should not update the comment for invalid user', async () => {
+    it("should only update the author's comment", async () => {
```

Comment:
```suggestion
    it('should only update the author's comment', async () => {
```

Enable format on save in vs code :)

Reply:
I have that enabled, but with `\'s` it automatically results to the double quotes

Reply:
NVM. I didn't realize that there was an apostrophe. Ignore me